app/plugins/datadog.md

```diff
@@ -44,6 +44,7 @@ parameter                      | default     | description
 `config.port`<br>*optional*    | `8125`      | The port to send data to on the upstream server.
 `config.metrics`<br>*optional* | All metrics<br>are logged | List of Metrics to be logged. Available values are described at [Metrics](#metrics).
 `config.prefix`<br>*optional* | `kong` | String to be attached as prefix to metric's name.
+`config.tag_api_name`<br>*optional* | `false` | Whether to tag every metric with the api's name. If set to true then a tag of `api_name:<api-name>` will be included with metrics sent to datadog.
```

Comment:
I think you should also mention that if `config.tag_api_name` is `true`, API name will not be prefixed in metrics identifier.

Reply:
Absolutely. I've updated it.